examples/general/abs_home_path/ui_test/src/main.rs

```diff
@@ -0,0 +1,28 @@
+fn main() {
+    println!("Testing abs_home_path lint's test context allowance");
+
+    // This SHOULD trigger the abs_home_path lint because we're NOT in a test function
+    non_test_function_with_home_path();
+}
+
+// This function should trigger the lint because it's not a test function
+fn non_test_function_with_home_path() {
+    // This should trigger the lint when fix is disabled
```

Comment:
I think it is sufficient to have just one comment here, rather than three.

Maybe just the last one, so that there is symmetry with `test_function_with_home_path`?